addon/services/asset-loader.js

```diff
@@ -244,6 +244,28 @@ export default Ember.Service.extend({
         link.href = uri;
 
         document.head.appendChild(link);
+
+        // In case the browser doesn't fire onload/onerror events, we poll the
+        // the list of stylesheets to see when it loads
+        function checkSheetLoad() {
+          const stylesheets = document.styleSheets;
```

Comment:
It's not needed for every browser, but there is no way to realiably feature detect the load/error events. Having it in every browser doesn't have any real drawbacks.

I don't think we want to "fix" render blocking, since we don't want to render until the style sheet is downloaded and parsed.